pkg/controller/command/command_controller.go

```diff
@@ -567,3 +625,81 @@ func (r *ReconcileCommand) getConfig(command *contrail.Command) (*contrail.Confi
 
 	return config, err
 }
+
+func (r *ReconcileCommand) getWebUI(command *contrail.Command) (*contrail.Webui, error) {
+	webUI := &contrail.Webui{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{
+		Namespace: command.GetNamespace(),
+		Name:      command.Spec.ServiceConfiguration.WebUIInstance,
+	}, webUI)
+
+	return webUI, err
+}
+
+func (r *ReconcileCommand) reconcileBootstrapJob(command *contrail.Command, commandBootStrapConfigName string) error {
+	bootstrapJob := &batch.Job{}
+	jobName := types.NamespacedName{Namespace: command.Namespace, Name: command.Name + "-bootstrap-job"}
+	err := r.client.Get(context.Background(), jobName, bootstrapJob)
+	alreadyExists := err == nil
```

Comment:
Shouldn't we check if error is not found?
In my opinion we should return err if error is different then in not found

Reply:
Done